hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastNamespaceHandler.java

```diff
@@ -40,6 +40,10 @@ public void init() {
                 "set",
                 "list",
                 "executorService",
+                "durableExecutorService",
+                "scheduledExecutorService",
+                "ringBuffer",
+                "cardinalityEstimator",
```

Comment:
it's weird it worked without it...  I created an issue for the other missing ones: https://github.com/hazelcast/hazelcast/issues/12121